lib/versioncontrol/upgradewindow/upgradewindow.go

```diff
@@ -313,6 +316,11 @@ type Driver interface {
 	// called if teleport experiences prolonged loss of auth connectivity, which may be an indicator
 	// that the control plane has been upgraded s.t. this agent is no longer compatible.
 	Reset(ctx context.Context) error
+
+	// ForceNOP sets the NOP schedule, ensuring that updates do not happen.
+	// This schedule was originally for testing, but now it also ensures that
+	// the teleport-update binary can disable all versions of the teleport-upgrader script.
+	ForceNop(ctx context.Context) error
```

Comment:
nit: ForceNOP, like the godoc?

Reply:
Will update the godoc to `ForceNop`. `Nop` is not an initialism, and the standard library uses this spelling (e.g., `io.NopCloser`.